recreate-db.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+CONTAINER_ID=$(docker ps -aqf "name=dr-trottoir-4-database-1")
+docker exec -it $CONTAINER_ID psql -U django -d postgres -c "DROP DATABASE drtrottoir WITH (FORCE);"
+docker exec -it $CONTAINER_ID psql -U django -d postgres -c "CREATE DATABASE drtrottoir;"
+./migrations.sh
+docker-compose exec backend-wsgi python manage.py loaddata datadump.json
```

Comment:
Here as well `backend-wsgi` ==> `backend`

Reply:
You are right, this is fixed now